### PR TITLE
use phidgets rather than simulated relayboard on raw3-1

### LINKED ISSUE #403: raw3-1 emergency state topic in wrong ns
in raw3-1.xml the relayboard.launch is started in simulation mode which results in a wrong ns of the /emergency_stop_state topic (which is relayboard/emergency_stop_state) and the robot can not be moved.

### PR DESCRIPTION
**Pull request by fmessmer**
(supposed to) fix #403 

This is just a copy of the raw3-3 configuration and needs to be adjusted and tested on raw3-1
@ipa-bnm can you help me here?

### CODE REVIEW

**Comment by fmessmer** [comment]
raw3-1 is out of the office...

**Comment by mgruhler** [comment]
Currently in the Gewerbestrasse....

File: cob_hardware_config/raw3-1/config/phidgets.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+frequency: 30                       #nodes update frequency in hz
+update_mode: polling                #switch between poll and event driven processing [polling, event]
+
+boards:
+  ifk_general:
+    serial_num: 285651
```
Comment (benmaidel) [inline cob_hardware_config/raw3-1/config/phidgets.yaml:6]: this needs to be adjusted to the real serial on raw3-1

File: cob_hardware_config/raw3-1/config/phidgets.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    ratiometric: true
+    sensors:
+      bat1:
```
Comment (benmaidel) [inline cob_hardware_config/raw3-1/config/phidgets.yaml]: change from `bat1:` to `voltage:`

File: cob_hardware_config/raw3-1/config/phidgets.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        type: analog
+        index: 0
+      bat2:
```
Comment (benmaidel) [inline cob_hardware_config/raw3-1/config/phidgets.yaml]: change from `bat2:` to `current:

File: cob_hardware_config/raw3-1/config/phidgets.yaml (hunk 1 above)
Comment (fmessmer) [inline cob_hardware_config/raw3-1/config/phidgets.yaml:6]: do you know the serial number of raw3-1?

Comment (benmaidel) [inline cob_hardware_config/raw3-1/config/phidgets.yaml:6]: nope, we need to start the driver on raw3-1 and check for the serial

**Pull request merged**